.github/workflows/prof_controller_intergration_test.yaml

```diff
@@ -0,0 +1,62 @@
+name: Profile Controller Intergration Test
+on:
+  pull_request:
+    paths:
+      - components/profile-controller/**
+      - components/access-management/**
+    branches:
+      - master
+      - v*-branch
+
+env:
+  PROFILE_IMG: profile-controller
+  KFAM_IMG: kfam
+  TAG: intergration-test
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Build Profile Controller Image 
+      run: |
+        cd components/profile-controller
+        make docker-build IMG=${{env.PROFILE_IMG}}
+
+    - name: Build Kfam Image
+      run: |
+        cd components/access-management
+        make docker-build IMG=${{env.KFAM_IMG}}
```

Comment:
Let's avoid building the `kfam` image here. Since in the next iterations https://github.com/kubeflow/kubeflow/issues/6766#issuecomment-1342769182 we'll be using the `latest` tag we should be able to use that.

Also currently we use the `v1.6.0` tag for kfam, so we don't need to build the image